edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -1,436 +1,326 @@
-    width: 100%;
-    border-radius: 3px;
-    margin: 5px 0 0 0;
-    padding: 5px 10px;
-    background: #D4EAE4;
-    list-style: none;
+    .action-more,
+    .entitlement-action-more {
+      width: 100%;
+      color: $gray;
+      background: $white;
+      border: none;
+      box-shadow: none;
+      text-shadow: none;
+      font-size: 0.75rem;
+    }
   }
 
-  .course .details .course-actions .wrapper-action-more
-  .actions-dropdown.is-visible{
-      display: block;
-      pointer-events: auto;
-  }
-  .course .details .course-actions .wrapper-action-more
-  .actions-dropdown .actions-dropdown-list {
-      box-sizing: border-box;
-    display: table;
-    box-shadow: 0 1px 1px rgba(0,0,0,0.1);
-    position: relative;
-    width: 100%;
-    border-radius: 3px;
-    margin: 5px 0 0 0;
-    padding: 5px 10px;
-    background: #D4EAE4;
-  }
+  // Messages.
+  .wrapper-messages-primary {
+    .messages-list {
+      background-color: $dashboard-contrast;
+    }
 
-  //dashboard-subcription-mobile
-  @include media-breakpoint-down(lg) {
-    .course .details {
-      padding-left: 10px;
+    .message {
+      padding: 15px;
     }
-  }
 
-  @include media-breakpoint-down(md) {
-    .course .details .wrapper-course-image {
+    .message-copy {
       display: none;
     }
 
-    .course .details .wrapper-course-details {
-      display: flex;
-      flex-direction: column;
+    // Upgrade button.
+    .action-upgrade-container {
+      float: none;
+      margin: 0;
     }
 
-    .course .details .wrapper-course-details .course-info,
-    .course .details .wrapper-course-details .wrapper-course-actions {
-      flex-grow: 1;
-      text-align: left;
+    .action-upgrade {
+      color: $gray;
+      background: none;
+      box-shadow: none;
+      padding: 0;
+
+      &:hover,
+      &:focus {
+        background: none;
+      }
+
+      span {
+        color: $gray;
+        font-weight: 400;
+        font-size: 0.875rem;
+      }
+
+      .action-upgrade-icon {
+        border: 1px solid $checkbox-border;
+
+        &:hover,
+        &:focus {
+          border: 1px solid $black;
+        }
+      }
     }
 
-    .course .details .wrapper-course-details
-    div.wrapper-course-actions div.course-actions {
-      padding-top: 50px;
+    // Verify button.
+    .verification-reminder,
+    .verification-cta {
+      display: none;
     }
 
-    .course .details .wrapper-course-details .course-info .wrapper-action-more,
-    .course .details .wrapper-course-details .course-info .entitlement-actions-wrapper,
-    .course .details .wrapper-course-details .wrapper-course-actions .wrapper-action-more,
-    .course .details .wrapper-course-details .wrapper-course-actions .entitlement-actions-wrapper {
-      padding-top: 3px;
+    .wrapper-extended {
+      padding: 0;
     }
   }
 
-  @include media-breakpoint-down(sm) {
-    min-width: 0;
+  // Item responsive layout.
+  @include media-breakpoint-up(md) {
+    width: 48%;
```

```suggestion
    width: 48%;
    margin: 0 2% 30px 0;
```

would you consider to apply this suggestion to stick more to the Figma file?